### PR TITLE
REL-1841: Update GRACES PIT options

### PR DESCRIPTION
**Pull request by cquiroz**
This PR contains two updates to the Graces model:

* Rename of fiber modes
* Addition of read mode

The PR takes care of the xml, model and decision tree changes plus tests

### CODE REVIEW

**Comment by cquiroz** [comment]
Comments?

**Comment by arturog8m** [comment]
Everything seems consistent at least to my eyes! :+1: Others can comment on the scala code in more detail, but looks fairly straightforward.

File: bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/dtree/inst/Graces.scala [hunk 1]
```diff
@@ -22,4 +21,19 @@ object Graces {
     }
   }
 
+  class ReadNode(fm: GracesFiberMode)  extends SingleSelectNode[GracesFiberMode, GracesReadMode, GracesBlueprint](fm) {
```
Comment (swalker2m) [inline bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/dtree/inst/Graces.scala]: Typo here, I presume.  Should be `ReadMode`.

**Comment by swalker2m** [comment]
Other than the typo, everything looks fine to me to.  :+1:

File: bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/dtree/inst/Graces.scala (hunk 1 above)
Comment (cquiroz) [inline bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/dtree/inst/Graces.scala]: Actually it refers to the Node on the decision tree, like the previous one is called `FiberNode`
Perhaps a better name is `ReadModeNode`

**Pull request merged**